app/admin/child.rb

```diff
@@ -42,4 +42,10 @@
   action_item :import_spreadsheet, only: :index do
     link_to 'Import Spreadsheet', new_spreadsheet_families_path if authorized?(:import, Family)
   end
+
+  controller do
+    def permitted_params
+      params.permit!
```

Comment:
This will permit all params which makes the call to `permit_params` on line 15 redundant. I think it would be better to use `permit_params` if possible, but maybe you had trouble getting it to work?

Reply:
This was left over from trying to fix an issue, it has now been removed and all the params have been added